pkg/destroy/alibabacloud/alibabacloud.go

```diff
@@ -1472,3 +1483,17 @@ func convertResourceArn(arn string) (resourceArn ResourceArn, err error) {
 	resourceArn.Arn = arn
 	return resourceArn, nil
 }
+
+func splitArray(arr []string, size int) [][]string {
+	var quantity = int(math.Ceil(float64(len(arr)) / float64(size)))
```

Comment:
in order to improve readability, perhaps 
```suggestion
	var numRows = int(math.Ceil(float64(len(arr)) / float64(size)))
```